composer.json

```diff
@@ -17,20 +17,9 @@
         }
     ],
     "require": {
-        "php" : ">=7.0",
-        "yubico/u2flib-server": "1.0.1",
-        "laravel/framework": "^5.8",
-        "illuminate/auth": "^5.8",
-        "illuminate/config": "^5.8",
-        "illuminate/container": "^5.8",
-        "illuminate/events": "^5.8",
-        "illuminate/database": "^5.8",
-        "illuminate/session": "^5.8",
-        "illuminate/routing": "^5.8",
-        "illuminate/http": "^5.8",
-        "illuminate/contracts": "^5.8",
-        "illuminate/view": "^5.8",
-        "illuminate/translation": "^5.8"
+        "php" : ">=7.2",
+        "laravel/framework": "^6.0",
```

Comment:
some illuminate packages are use in this package. The laravel/framework doesn't require any illuminate packages

[https://packagist.org/packages/laravel/framework](https://packagist.org/packages/laravel/framework)